specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json

```diff
@@ -2503,6 +2534,47 @@
         }
       },
       "description": "The response of the List MessagingRegions operation."
+    },
+	"MessagingPlanproperties": {
+      "properties": {
+        "sku": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Sku type"
+        },
+        "selectedEventHubUnit": {
+          "format": "int32",
+          "type": "integer",
+          "description": "Selected event hub unit"
+        },
+        "updatedAt": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The exact time the messaging plan was updated."
+        },
+        "revision": {
+          "format": "int64",
+          "type": "integer",
+          "description": "revision number"
+        }
+      },
+      "description": "Messaging Region"
```

Comment:
>Messaging Region [](start = 22, length = 16)

Please add better description of the model. I'm not clear how this model even relates to regions. The only exposed operation in the swagger is "Get" on Messaging plan which suggests that there should be only one Messaging plan per namespace. Did you miss "List" operation?